### ISSUE TITLE
Issue with Error Messages and 'WhisperModel' Attribute in Code

### ISSUE THREAD

**Issue by hcljsq** [collabora/WhisperLive#88]
Could you please help explain what these error messages mean and how to resolve them? Thank you.

```bash
ERROR:root:[ERROR]: sent 1000 (OK); then received 1000 (OK)
ERROR:root:received 1001 (going away); then sent 1001 (going away)
ERROR:root:[ERROR]: 'WhisperModel' object has no attribute 'model'
ERROR:root:received 1001 (going away); then sent 1001 (going away)
ERROR:root:[ERROR]: 'WhisperModel' object has no attribute 'model'
ERROR:root:no close frame received or sent
ERROR:root:[ERROR]: 'WhisperModel' object has no attribute 'model'
```

**Comment by zoq** [comment]
Are you using the latest release from pip?

**Comment by hcljsq** [comment]
Yes, I was using `v0.0.10` from pip. What other information should I provide?

**Comment by makaveli10** [comment]
These are just debug messages when a websocket connection is ended/closed, i'll get rid of these and have better connection close message to avoid any confusion. You should have it in the upcoming release. Thanks!

**Comment by hcljsq** [comment]
And these errors:

```bash
ERROR:root:no close frame received or sent
ERROR:root:[ERROR]: 'WhisperModel' object has no attribute 'model'
ERROR:root:no close frame received or sent
ERROR:root:[ERROR]: 'WhisperModel' object has no attribute 'model'
ERROR:websockets.server:connection handler failed
Traceback (most recent call last):
  File "/home/ubuntu/.local/lib/python3.10/site-packages/websockets/sync/server.py", line 499, in conn_handler
    handler(connection)
  File "/home/ubuntu/WhisperLive/whisper_live/server.py", line 99, in recv_audio
    client = ServeClient(
  File "/home/ubuntu/WhisperLive/whisper_live/server.py", line 255, in __init__
    self.websocket.send(
  File "/home/ubuntu/.local/lib/python3.10/site-packages/websockets/sync/connection.py", line 284, in send
    with self.send_context():
  File "/usr/lib/python3.10/contextlib.py", line 135, in __enter__
    return next(self.gen)
  File "/home/ubuntu/.local/lib/python3.10/site-packages/websockets/sync/connection.py", line 724, in send_context
    raise self.protocol.close_exc from original_exc
websockets.exceptions.ConnectionClosedOK: received 1001 (going away); then sent 1001 (going away)
WARNING:root:<whisper_live.server.ServeClient object at 0x7ff0042e7e80> Client disconnected due to overtime.
ERROR:root:[ERROR]: 'WhisperModel' object has no attribute 'model'
ERROR:websockets.server:connection handler failed
Traceback (most recent call last):
  File "/home/ubuntu/.local/lib/python3.10/site-packages/websockets/sync/server.py", line 499, in conn_handler
    handler(connection)
  File "/home/ubuntu/WhisperLive/whisper_live/server.py", line 99, in recv_audio
    client = ServeClient(
  File "/home/ubuntu/WhisperLive/whisper_live/server.py", line 227, in __init__
    self.transcriber = WhisperModel(
  File "/home/ubuntu/WhisperLive/whisper_live/transcriber.py", line 132, in __init__
    self.model = ctranslate2.models.Whisper(
RuntimeError: CUDA failed with error out of memory
```

If you can give me some hints, I would like to make create PRs!

**Comment by makaveli10** [comment]
@hcljsq can you test this PR if this resolves your issue
https://github.com/collabora/WhisperLive/pull/94

**Comment by hcljsq** [comment]
Yes, I will test it within 24 hours. Thank you.

**Comment by hcljsq** [comment]
I found that this Pull Request was merged, can you also release a new pip version? So I can test it. Thank you.

**Comment by makaveli10** [comment]
@hcljsq released a new version `v0.0.11`. You can upgrade and test. Thanks

**Comment by makaveli10** [comment]
Closing due to inactivity. Feel free to re-open if issue persists.
